### PR TITLE
LINUX_DEFCONFIG parse if defined or use default values

### PR DESCRIPTION
**Pull request by hyphop**
this change for next building different kernel variants

### CODE REVIEW

**Comment by hyphop** [comment]
i have we must have this one
LINUX_DEFCONFIG=another_config make kernel
or chouse LINUX_DEFCONFIG via source env 
for example i have prepare light config already for kernel which will be best for some distros

File: config/boards/Edge.conf [hunk 1]
```diff
@@ -31,7 +31,7 @@ case "$LINUX" in
 		LINUX_DTB="arch/arm64/boot/dts/rockchip/rk3399-khadas-edge-linux.dtb"
 		LINUX_DIR="$ROOT/linux"
 		LINUX_GIT_BRANCH="khadas-edge-4.4.y"
-		LINUX_DEFCONFIG="kedge_defconfig"
+		LINUX_DEFCONFIG="${LINUX_DEFCONFIG:- kedge_defconfig}"
```
Comment (numbqq) [inline config/boards/Edge.conf]: You want to use other configurations for linux building?

File: scripts/create_image.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@ source config/functions/functions
 ## Try to update Fenix
 check_update() {
 	cd $ROOT
-
+	[ "$FENIX_CHECK_UPDATE_NO" ] && return
```
Comment (numbqq) [inline scripts/create_image.sh]: How to use `FENIX_CHECK_UPDATE_NO` ?

File: config/boards/Edge.conf (hunk 1 above)
Comment (hyphop) [reply config/boards/Edge.conf]: yes ! for some reasons we must have something like this ;)

File: scripts/create_image.sh (hunk 2 above)
Comment (hyphop) [reply scripts/create_image.sh]: FENIX_CHECK_UPDATE_NO=1 ./make

Comment (hyphop) [reply scripts/create_image.sh]: for example ! if i make some local changes and try to build and check how its woks or not

Comment (hyphop) [reply scripts/create_image.sh]: if we have another simple way plz write me ;)

Comment (numbqq) [reply scripts/create_image.sh]: You can just execute `touch .ignore-update` to ignore update.

Comment (hyphop) [reply scripts/create_image.sh]: ok ! this one what i need ;) tnx

File: config/boards/Edge.conf (hunk 1 above)
Comment (hyphop) [reply config/boards/Edge.conf]: ???

**Comment by numbqq** [comment]
Cherry picked.

**Pull request closed without merging**